pyproject.toml

```diff
@@ -43,7 +43,7 @@ sc-controller         = "scc.bin.sc_controller:main"
 
 [project.scripts]
 scc-daemon            = "scc.bin.scc_daemon:main"
-#scc                   = "scc.bin.scc:main"
```

Comment:
Probably just toss scc into the bin directory with everything else if we're re-enabling it, hopefully that doesn't somehow break it.

Reply:
Done

Reply:
That ended up going from `scripts/` to `scc/` instead of `scc/bin/`, and I recall the files not wanting to work without a .py suffix, could you toss it in the bin directory instead and see if it still works with .py?

If so, editors will pick it up as a Python file more easily.

Reply:
The ci build [succeeded](https://github.com/git-developer/sc-controller/actions/runs/11167315967), so `scc` should be working.